contracts/Removal.sol

```diff
@@ -23,22 +23,29 @@ contract Removal is
 {
   using SafeMathUpgradeable for uint256;
 
-  struct Vintage {
-    address supplier;
-    uint16 vintage;
-    // todo: location
-    // todo: methodology
-    // todo: supplier name
-  }
-
```

Comment:
I think this is safe to get rid of for now because all of this information (and more) is encoded in the token id.  If we end up in a place where we need more information for a removal, maybe we add back a struct or something else like a registry approach